build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id 'org.jetbrains.intellij' version '0.5.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.10'
+}
+
+group 'eu.gillissen.rider'
+version '0.2.4'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib"
+}
+
+// See https://github.com/JetBrains/gradle-intellij-plugin/
+intellij {
+    type 'RD'
+    version '2020.2'
```

Comment:
Either a stable version or an EAP snapshot (e.g. `2020.3-EAP3-SNAPSHOT`

Will be used as min version for the plugin, more docs at https://github.com/JetBrains/gradle-intellij-plugin/